lib/cisco_node_utils/bgp_af.rb

```diff
@@ -98,21 +98,41 @@ def set_args_keys(hash={}) # rubocop:disable Style/AccessorMethodName
     #                      PROPERTIES                      #
     ########################################################
 
+    # IOS XR?
+    def ios_xr?
```

Comment:
I like this - consider moving it into the NodeUtil class so it can be shared to other provider classes.